lib/push.js

```diff
@@ -25,89 +25,35 @@ var PUSH_COMMANDS = {
-  accountVerify: {
-    send: 'push.account_verify.send',
-    success: 'push.account_verify.success',
-    resetSettings: 'push.account_verify.reset_settings',
-    failed: 'push.account_verify.failed',
-    noCallback: 'push.account_verify.no_push_callback',
-    noKeys: 'push.account_verify.data_but_no_keys'
-  },
-  accountConfirm: {
-    send: 'push.account_confirm.send',
-    success: 'push.account_confirm.success',
-    resetSettings: 'push.account_confirm.reset_settings',
-    failed: 'push.account_confirm.failed',
-    noCallback: 'push.account_confirm.no_push_callback',
-    noKeys: 'push.account_confirm.data_but_no_keys'
-  },
-  passwordReset: {
-    send: 'push.password_reset.send',
-    success: 'push.password_reset.success',
-    resetSettings: 'push.password_reset.reset_settings',
-    failed: 'push.password_reset.failed',
-    noCallback: 'push.password_reset.no_push_callback',
-    noKeys: 'push.password_reset.data_but_no_keys'
-  },
-  passwordChange: {
-    send: 'push.password_change.send',
-    success: 'push.password_change.success',
-    resetSettings: 'push.password_change.reset_settings',
-    failed: 'push.password_change.failed',
-    noCallback: 'push.password_change.no_push_callback',
-    noKeys: 'push.password_change.data_but_no_keys'
-  },
-  deviceConnected: {
-    send: 'push.device_connected.send',
-    success: 'push.device_connected.success',
-    resetSettings: 'push.device_connected.reset_settings',
-    failed: 'push.device_connected.failed',
-    noCallback: 'push.device_connected.no_push_callback',
-    noKeys: 'push.device_connected.data_but_no_keys'
-  },
-  deviceDisconnected: {
-    send: 'push.device_disconnected.send',
-    success: 'push.device_disconnected.success',
-    resetSettings: 'push.device_disconnected.reset_settings',
-    failed: 'push.device_disconnected.failed',
-    noCallback: 'push.device_disconnected.no_push_callback',
-    noKeys: 'push.device_disconnected.data_but_no_keys'
-  },
-  profileUpdated: {
-    send: 'push.profile_updated.send',
-    success: 'push.profile_updated.success',
-    resetSettings: 'push.profile_updated.reset_settings',
-    failed: 'push.profile_updated.failed',
-    noCallback: 'push.profile_updated.no_push_callback',
-    noKeys: 'push.profile_updated.data_but_no_keys'
-  },
-  devicesNotify: {
-    send: 'push.devices_notify.send',
-    success: 'push.devices_notify.success',
-    resetSettings: 'push.devices_notify.reset_settings',
-    failed: 'push.devices_notify.failed',
-    noCallback: 'push.devices_notify.no_push_callback',
-    noKeys: 'push.devices_notify.data_but_no_keys'
-  },
-  accountDestroyed: {
-    send: 'push.account_destroyed.send',
-    success: 'push.account_destroyed.success',
-    resetSettings: 'push.account_destroyed.reset_settings',
-    failed: 'push.account_destroyed.failed',
-    noCallback: 'push.account_destroyed.no_push_callback',
-    noKeys: 'push.account_destroyed.data_but_no_keys'
+const reasonToEvents = (() => {
```

Comment:
nit: good refactor, can we rename this to make it verbose, maybe`reasonToEvents` -> `pushReasonsToEvents` ?